dpnp/tests/test_nanfunctions.py

```diff
@@ -122,6 +122,10 @@ def test_allnans(self, dtype, array):
 
         result = getattr(dpnp, self.func)(ia)
         expected = getattr(numpy, self.func)(a)
+        if array.shape == ():
+            # for "0d" case, dpnp returns 0D array, numpy returns 1D array
```

Comment:
The new version of specification proposes to leave 0d input as unspecified and thus implementation-defined (see [851](https://github.com/data-apis/array-api/pull/851)).
Thus it is not completely correct to state that numpy breaks the array API here.